pkg/validation/limits.go

```diff
@@ -60,9 +60,10 @@ type Limits struct {
 	QuerySplitDuration model.Duration `yaml:"split_queries_by_interval" json:"split_queries_by_interval"`
 
 	// Ruler defaults and limits.
-	RulerEvaluationDelay        model.Duration `yaml:"ruler_evaluation_delay_duration" json:"ruler_evaluation_delay_duration"`
-	RulerMaxRulesPerRuleGroup   int            `yaml:"ruler_max_rules_per_rule_group" json:"ruler_max_rules_per_rule_group"`
-	RulerMaxRuleGroupsPerTenant int            `yaml:"ruler_max_rule_groups_per_tenant" json:"ruler_max_rule_groups_per_tenant"`
+	RulerEvaluationDelay          model.Duration `yaml:"ruler_evaluation_delay_duration" json:"ruler_evaluation_delay_duration"`
+	RulerMaxRulesPerRuleGroup     int            `yaml:"ruler_max_rules_per_rule_group" json:"ruler_max_rules_per_rule_group"`
+	RulerMaxRuleGroupsPerTenant   int            `yaml:"ruler_max_rule_groups_per_tenant" json:"ruler_max_rule_groups_per_tenant"`
+	RulerRemoteWriteQueueCapacity int            `yaml:"ruler_remote_write_queue_capacity" json:"ruler_remote_write_queue_capacity"`
```

Comment:
@cyriltovena I know we discussed setting a glad for this new limit, but I've added one above in [config.go](https://github.com/grafana/loki/pull/3766/files#diff-3130844bd4888ce4a8ca873025331c32bc2c26aaee491f1a1211a31548ba4bd4R58) which defines the same thing; how should I proceed?

Reply:
If you want to allow overrides per Tenant then you should use the one in Limits and remove the other one. Otherwise if you want the same value for all tenants you can remove it here.

Reply:
Ah, I understand; thought I needed both. Thanks!